saleor/graphql/product/filters.py

```diff
@@ -298,9 +298,6 @@ class ProductFilter(django_filters.FilterSet):
     collections = GlobalIDMultipleChoiceFilter(method=filter_collections)
     categories = GlobalIDMultipleChoiceFilter(method=filter_categories)
     has_category = django_filters.BooleanFilter(method=filter_has_category)
-    price = ObjectTypeFilter(
```

Comment:
Although we remove this field from the product model, I think it would be still useful to be able to filter products by price. We could rewrite the filter logic to use variant prices instead.